moveit_plugins/moveit_ros_control_interface/moveit_ros_control_interface_plugins.xml

```diff
@@ -4,6 +4,11 @@
         <description></description>
     </class>
   </library>
+  <library path="moveit_ros_control_interface_empty_plugin">
+    <class name="admittance_controller/AdmittanceController" type="moveit_ros_control_interface::EmptyControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
```

Comment:
Also, a small irritation that slipped through the original review. Giving the name `admittance_controller/AdmittanceController` to an empty placeholder plugin is highly confusing.